src/main/java/com/gd/clinic/repository/PatientRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.gd.clinic.repository;
+
+import com.gd.clinic.entity.Patient;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Repository
```

Comment:
I see other repos don't have this annotation, so lets remove it from here or add everywhere

Reply:
Missed that one, didn't have issues because the classes extend `JpaRepository` but added the annotation to the other repository classes this commit.